spring-boot-project/spring-boot-actuator/src/test/java/org/springframework/boot/actuate/metrics/export/prometheus/PrometheusScrapeEndpointIntegrationTests.java

```diff
@@ -108,12 +108,12 @@ void scrapePrefersToProduceOpenMetrics100(WebTestClient client) {
 	@WebEndpointTest
 	void scrapeWithIncludedNames(WebTestClient client) {
 		client.get()
-			.uri("/actuator/prometheus?includedNames=counter1_total,counter2_total")
+			.uri("/actuator/prometheus?includedNames=counter1,counter2")
```

Comment:
This is a difference between simpleclient and the latest 1.x versions. Even though the output will have `counter1_total`, calls to `scrape` should filter with the name without the suffix added by Prometheus.